### DISCUSSION TITLE
Care if I try to sell one?

### DISCUSSION THREAD

**Discussion by tmanternach** [doudar/SmartSpin2k#374] [General]
I built a SmartSpin2k earlier this summer, it's been working great. Thanks for this amazing project.

I have enough left over parts (and a 3D printer) to build one more unit. Do you care if I try to sell one on the [Schwinn IC4 subreddit](https://www.reddit.com/r/SchwinnIC4_BowflexC6/)?

**Comment by doudar** [comment]
> I built a SmartSpin2k earlier this summer, it's been working great. Thanks for this amazing project.
> 
> 
> 
> I have enough left over parts (and a 3D printer) to build one more unit. Do you care if I try to sell one on the [Schwinn IC4 subreddit](https://www.reddit.com/r/SchwinnIC4_BowflexC6/)?

Not at all! That's totally in the spirit of the project. In fact if you'd like to build more, we can get you added to the kits section of the wiki. There's probably also a demand for just electronics or printed parts alone if that's your specialty.

Have fun!
